### PR TITLE
Added a Dockerfile for the mitm proxy

### PR DESCRIPTION
**Pull request by AngryKiller**
This pull request adds a Dockerfile for the mitm proxy (mitm.js) to deploy it easily on an external server using Docker. I tested it on macOS with Docker Desktop and on Debian 10, it builds and runs perfectly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +40,8 @@ root certificate at `certs/certs/ca.crt`
 On MacOS, you can trust the root certficate with one command: `sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain certs/certs/ca.crt`
 
 This should be perfectly safe.
+
+# Docker image
+
+To build an image for the MITM proxy, simply run `npm run docker-mitm`, and then create a container binding the port 8082 and the folder `/usr/src/app/certs` to the host to get access to the certificates.
```
Comment (AnanthVivekanand) [inline README.md:46]: An explanation of the binding process / importing certs would be appreciated.

**Comment by AnanthVivekanand** [comment]
To trust the root CA, did you have to manually copy over the certs from `/usr/src/app/certs` to the host? I'm not too familiar with Docker,  `/usr/src/app/certs`  is within the container, correct?

**Comment by AngryKiller** [comment]
> To trust the root CA, did you have to manually copy over the certs from `/usr/src/app/certs` to the host? I'm not too familiar with Docker, `/usr/src/app/certs` is within the container, correct?

With Docker, you can bind folders that are inside the container to a path in the host machine. For example, if you bind the folder /usr/src/app/certs of the container to /opt/spotify-adblock/certs, you will be able to access the contents of /usr/src/app/certs in the path /opt/spotify-adblock/certs of the host machine. And for port binding, you basically tell Docker that you want the port 8082 inside the container to be externally reachable with whatever port you want. I will add an example command to run the container in the README if you want to.

**Comment by AnanthVivekanand** [comment]
Yes, could you add an example for the path binding in the README so that certs inside the container can be easily trusted / added to the keychain of the host machine? Much appreciated, thanks for this work.

**Comment by AngryKiller** [comment]
I added an example command to run the proxy with Docker in the readme.

**Comment by AnanthVivekanand** [comment]
Thanks!

**Pull request merged**